client/src/components/ProtectedRoute.js

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import { Route, Redirect } from "react-router-dom";
+
+const ProtectedRoute = ({ component: Component, ...rest }) => {
+  const checkLocalStorage = localStorage.getItem("isAuthenticated");
+  return (
```

Comment:
Hey I think you should also check the JWT time stamp. Because they do expire after a certain time period.

Reply:
As discussed on our call, the JWT time stamped is handled on the backend. resolving